build.gradle

```diff
@@ -26,12 +26,14 @@ plugins {
 }
 
 repositories {
+  mavenLocal()
   mavenCentral()
 }
 
 ext {
   lombokVersion = '1.18.8'
   slf4jVersion = '1.7.26'
+  isReleaseVersion = !version.endsWith("SNAPSHOT")
```

Comment:
I like the approach of defining a flag for development versions! But, is `version` a global variable? I don't see it defined.